File: Component/BusiLib/src/main/java-person/com/component/person/holder/EmptyProducationHolder.java
package com.component.person.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.component.busilib.R;

public class EmptyProducationHolder extends RecyclerView.ViewHolder {
    ImageView mEmptyImg;
    TextView mEmptyTxt;

    public EmptyProducationHolder(View itemView) {
        super(itemView);
        mEmptyImg = (ImageView) itemView.findViewById(R.id.empty_img);
        mEmptyTxt = (TextView) itemView.findViewById(R.id.empty_txt);
    }

    public void bindData(boolean isSelf) {
        if (isSelf) {
            mEmptyTxt.setText("可以去练歌房录制作品哦～");
        } else {
            mEmptyTxt.setText("ta还没有作品哦～");
        }
    }
}

File: Component/BusiLib/src/main/java-person/com/component/person/producation/holder/EmptyProducationHolder.kt
package com.component.person.producation.holder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.component.busilib.R

class EmptyProducationHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    internal var mEmptyImg: ImageView
    internal var mEmptyTxt: TextView

    init {
        mEmptyImg = itemView.findViewById<View>(R.id.empty_img) as ImageView
        mEmptyTxt = itemView.findViewById<View>(R.id.empty_txt) as TextView
    }

    fun bindData(isSelf: Boolean) {
        if (isSelf) {
            mEmptyTxt.text = "可以去练歌房录制作品哦～"
        } else {
            mEmptyTxt.text = "ta还没有作品哦～"
        }
    }
}
